content/docs/esc/integrations/infrastructure/pulumi-iac/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+
+title: Pulumi IaC
+title_tag: Pulumi ESC and IaC integration
+h1: ESC Pulumi IaC Integration
+meta_desc: Pulumi ESC integrates with Pulumi IaC to expose environment settings and secrets to Pulumi stacks, simplifying configuration management.
+menu:
+  pulumiesc:
+    identifier: esc-pulumi-iac-integrations
+    parent: esc-inf-tools-integrations
+    weight: 1
+---
+
+## Using environments with Pulumi IaC
```

Comment:
I don't think we need a separate heading here. We should just make this the title/H1 of the page.

Reply:
done